src/chain_sync/sync_status.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright 2019-2025 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use crate::blocks::TipsetKey;
+use crate::lotus_json::lotus_json_with_self;
+use crate::networks::calculate_expected_epoch;
+use crate::shim::clock::ChainEpoch;
+use crate::state_manager::StateManager;
+use chrono::{DateTime, Utc};
+use fvm_ipld_blockstore::Blockstore;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::sync::Arc;
+use tracing::log;
+
+// Node considered synced if the head is within this threshold.
+const SYNCED_EPOCH_THRESHOLD: u64 = 5;
```

Comment:
Hm, that's 2m30s for calibnet and mainnet. Did lower values give you trouble?

Reply:
I didn't see any problem with this, I was experimenting with different numbers to see if there could be any issue but didn't. So just choose 5, since last was 10 (@LesnyRumcajs was there any specific reason for choosing 10?)

Reply:
I don't remember exactly. Lower values might have given flaky results in the healthcheck endpoint, i.e., frequent transitions between healthy and unhealthy due to forks. I'm okay with both.

Reply:
Let's revert to limit the changes. If someone reports that the difference is too big, we can always revisit this.

Reply:
Done.